about.md

```diff
@@ -49,6 +49,6 @@ Logo by [Marlo Yarlo](https://www.instagram.com/marloyarlo/).
 
 ASCII drawings adapted from original work by [Joan G. Stark](https://en.wikipedia.org/wiki/Joan_Stark) (a.k.a. jgs, Spunk) and Adel Faure.
 
-Headings: [Hershey-Futural](https://gitlab.com/swrs/hershey-futural), a typeface designed by Dr. A.V. Hershey  in the late 60s. Originally designed to be rendered using vectors on early cathode ray tube displays. SIL Open Font License.
+Headings: [Hershey-Futural](https://gitlab.com/swrs/hershey-futural), a typeface designed by Dr. A.V. Hershey in the late 60s. Originally designed to be rendered using vectors on early cathode ray tube displays. SIL Open Font License.
 
-ASCII drawings: [jgs](https://gitlab.com/velvetyne/jgs/-/tree/main), designed by Adel Faure as a tribute to jgs. This font has been specifically designed to draw ASCII art. SIL Open Font License. 
\ No newline at end of file
+ASCII drawings: [jgs](https://gitlab.com/velvetyne/jgs/-/tree/main), designed by Adel Faure as a tribute to jgs. This font has been specifically designed to draw ASCII art. SIL Open Font License.
```

Comment:
Should we add BC Sans here?

Reply:
Added in [ccb737](https://github.com/ournetworks/ournetworks.ca/pull/287/commits/3ccb73717ea2c69e85197ede7a185e9b62110ce8)